code_of_conduct.md

```diff
@@ -47,6 +48,7 @@ The following behaviors are considered harassment and are unacceptable within ou
 *   Deliberate intimidation, stalking or following (online or in person).
 *   Advocating for, or encouraging, any of the above behavior.
 *   Sustained disruption of community events, including talks and presentations.
+*   Collecting or sharing personal information (including names, emails, nicks, messages, emails, or other biographical or identifying information) without explicit and prior consent.
```

Comment:
This is very broad and includes logging or just caching.

Reply:
The Code of Conduct applies to people communicating over DES media and venues, as well as attending DES supported events, not DES itself, so I believe the society could still operate our own logs, etc. Specifically, we're trying to prohibit people from signing up and mining the Slack channels as we saw that happen on a few other Slacks here in town.

Any suggestions on making this clearer?

In terms of DES' use of the data, a privacy policy would probably be the place to do that. I've setup #19 to remember to do this.

Reply:
Should "nicks" be "nicknames"